scripts/servers.py

```diff
@@ -722,3 +722,50 @@ def managed_webdriverio_server(
             yield proc
     finally:
         del os.environ['MOBILE']
+
+
+@contextlib.contextmanager
+def managed_acceptance_tests_server(
+    suite_name: str = 'full',
+    sharding_instances: int = 1,
+    chrome_version: Optional[str] = None,
+    stdout: int = subprocess.PIPE,
+) -> Iterator[psutil.Process]:
+    """Returns context manager to start/stop the Acceptance server gracefully.
+
+    Args:
+        suite_name: str. The suite name whose tests should be run. If the value
+            is `full`, all tests will run.
+        sharding_instances: int. How many sharding instances to be running.
+        chrome_version: str|None. The version of Google Chrome to run the tests
+            on. If None, then the currently-installed version of Google Chrome
+            is used instead.
+        stdout: int. This parameter specifies the executed program's standard
+            output file handle.
+
+    Yields:
+        psutil.Process. The webdriverio process.
+
+    Raises:
+        ValueError. Number of sharding instances are less than 0.
+    """
+    if sharding_instances <= 0:
```

Comment:
Do you actually need all this stuff for acceptance tests?

Please drop everything that is not relevant or used, so that we can keep the code as simple as possible.